xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -1294,6 +1294,7 @@ void CVideoPlayer::Process()
       CSingleLock lock(m_StateSection);
       if (m_displayLost)
       {
+        CSingleExit unlock(m_StateSection);
```

Comment:
CSingleExit makes no sense here. Why did you ignore my advice to drop the lock completely?